platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -0,0 +1,159 @@
+import { api } from 'dicomweb-client';
+import DICOMWeb from '../../../DICOMWeb/';
+import RetrieveMetadataLoader from './retrieveMetadataLoader';
+import { sortStudySeries, sortingCriteria } from '../../sortStudy';
+import getSeriesInfo from '../../getSeriesInfo';
+import * as StudyUtils from '../../studyUtils';
+
+/**
+ * Search series of a given study
+ * @param {DICOMwebClient} client Dicomweb client api
+ * @param {string} studyInstanceUID The Study Instance UID to search series from;
+ * @returns {Arrays} A list of Series Instances
+ */
+async function searchStudySeries(client, studyInstanceUID) {
+  const seriesList = await client.searchForSeries({ studyInstanceUID });
```

Comment:
What data is fetched from this? Is there anyway to apply our series specific filter with this request?

Reply:
ok changed it a bit.. using api to fetch series (filtered)